### PR TITLE
ui: set correct font for name column in models list

### PR DESCRIPTION
**Pull request by feloy**
### What does this PR do?

Change the font of the first column in the models list, to have the same as in other lists

### What issues does this PR fix or reference?

Part of #479

### CODE REVIEW

**Review by axel7083 — APPROVED** [review]

File: packages/frontend/src/lib/table/model/ModelColumnName.svelte [hunk 1]
```diff
@@ -8,6 +8,8 @@ function openDetails() {
 }
 </script>
 
-<button class="text-sm text-gray-700" on:click="{() => openDetails()}">
-  {object.name}
+<button on:click="{() => openDetails()}">
+  <div class="text-sm text-gray-300 overflow-hidden text-ellipsis">
```
Comment (lstocchi) [inline packages/frontend/src/lib/table/model/ModelColumnName.svelte:12]: Should this be white? 
By looking at the color here https://design.penpot.app/#/view/52961d58-0a92-80c2-8003-2a6919ad37d6?page-id=51af7f49-bd09-8090-8003-8c5efc23847f&section=interactions&index=12 it seems to me the name is white and the rest is gray-300

Comment (feloy) [reply packages/frontend/src/lib/table/model/ModelColumnName.svelte:12]: I find difficult to extract the color from the mockup.

I reused the one used in other tables (AI Apps, Pods, etc)

Comment (feloy) [reply packages/frontend/src/lib/table/model/ModelColumnName.svelte:12]: Not sure if it is intentional to have different colors in this specific table, compared to other tables.

Should I 100% respect the mockup (white for first column, gray-300 for other columns), or have a consistent design across tables (gray-300 for first column, gray-700 for others)? @ekidneyrh WDYT?

Comment (feloy) [reply packages/frontend/src/lib/table/model/ModelColumnName.svelte:12]: Do we want to make this change of colors across all pages in Podman Desktop too? Or only on AI Studio?

If only on AI Studio, I can do it as part of this PR. If it's across Podman Desktop, I would prefer to sync with PD as part of another PR

**Review by lstocchi — APPROVED** [review]
LGTM

**Pull request merged**